components/sandbox/index.js

```diff
@@ -142,6 +142,9 @@ class Sandbox extends Component {
 				width: 100%;
 				height: 100%;
 			}
+			body > div > iframe {
```

Comment:
Isn't the above `body.video > div > iframe` now redundant with this?

Reply:
Hi @aduth ,
> Isn't the above body.video > div > iframe now redundant with this?

The part that applies width is, but the rule also sets height for videos and is not applying just to the equivalent selector (body > div > iframe / body.video > div > iframe) but also to other selectors. The equivalent rules without redundancy would be:
```
			body.video,
			body.video > div,
			body.video > div > iframe {
				height: 100%;
			}

			body.video,
			body.video > div,
			body > div > iframe {
				width: 100%;
			}
```
or
```
			body.video,
			body.video > div,
				width: 100%;
				height: 100%;
			}
			body.video > div > iframe {
				height: 100%;
			}
			body > div > iframe {
				width: 100%;
			}
```

Both alternatives seem worse compared to what we have so preferred to keep selectors for video different from the general ones.

Reply:
Oh, right, I missed the other style.

Still, it seems odd that there's overlap, with some styles targeting video specifically, if it's more of a general issue with iframes in embedded content.